### PR TITLE
adding roy

### PR DESCRIPTION
**Pull request by roycboyc**

### CODE REVIEW

**Comment by sahuguet** [comment]
Tests failed.
Fix your code and re-submit.
You can only have one emoji.

**Comment by ghost** [comment]
request failed please change and resubmit or close.

**Comment by ghost** [comment]
Might fail again since, the json is invalid. you missed a comma.

**Comment by roycboyc** [comment]
just fixed it -- sorry for that. thanks, Akshay!

File: _data/fall_2016/roycboyc.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "emoji": "brifcase"
+  "emoji": ":brifcase:",
```
Comment (ghost) [inline _data/fall_2016/roycboyc.json]: "brifcase" is misspelled. Also you don't need ": :"
E.g. https://github.com/startup-systems/students/pull/44/files

**Comment by ghost** [comment]
Wow you seem to have really bad luck, somehow the emoji you picked isn't allowed.

**Comment by roycboyc** [comment]
omg... sorry about this, guys. changed it again.

**Comment by ghost** [comment]
This time the error was due to another student.

**Pull request merged**